### ISSUE TITLE
Implement GPG for signed commits

### ISSUE THREAD

**Issue by gustavoggsb** [felipecrs/dotfiles#27]
Taking into account that we can work with teams, and that we have to ensure that the owner of the commit is really the person behind it, we need a way to sign these commits. One of these ways is to use a GPG key.

It would be interesting to have a way to facilitate this configuration, so that it does not become tiring to redo this whenever we start a new Linux.

**Comment by felipecrs** [comment]
I agree. I'll think about it. I'm just reluctant to introduce the need of interactivity to this dotfiles because it installs on Codespaces, which is not interactive at all.

**Comment by felipecrs** [comment]
> I don't know very well how to proceed here. Perhaps, by adding the key using chezmoi's encryption mechanism? I would also need to add a note in README for advising users forking the repository.

From https://github.com/felipecrs/dotfiles/pull/28
